src/model/model.nit

```diff
@@ -2602,3 +2615,63 @@ fun interface_kind: MClassKind do return once new MClassKind("interface", false)
 fun enum_kind: MClassKind do return once new MClassKind("enum", false)
 # The class kind `extern`
 fun extern_kind: MClassKind do return once new MClassKind("extern class", false)
+
+# Mokup constructed model used to test various model-related methods
```

Comment:
I agree, this is not a mock; however, I don't thinks it is a fake or a stub. As all of these terms denote *double* objets, ie objects that you use instead of the real ones. Here I use real objects --- I'm a classic TDDer :)

Might *standalone* be a better term?